.env

```diff
@@ -60,6 +60,7 @@ BAHMNI_OBSCALCULATOR_PATH
 BAHMNI_APPS_PATH
 ODOO_EXTRA_ADDONS
 ODOO_CONFIG_PATH
+METABASE_CONFIG_PATH
```

Comment:
Since this will be packaged in Bahmni Config as a metabase/ folder [here](https://github.com/mekomsolutions/bahmni-config-haiti/tree/master/configuration), we'll directly use BAHMNI_CONFIG_PATH